src/components/ContributorThumbnail.js

```diff
@@ -235,7 +238,7 @@ const Thumbnail = ({
             >
               {organization.name ? organization.name : organization}
             </Link>{' '}
-            {dropdownLength ? `(${dropdownLength})` : ` `}
+            {dropdownLength ? checkboxValue || (inputValue !== undefined && inputValue !== "") ? `( ${dropdownLength} / ${organization.totalCount})` : `(${dropdownLength})` : ` `}
```

Comment:
Might want to put this logic into a small function and then call the function here.  That way you can use if/else statements instead of so many "?" operators